lab_2_tokenize_by_bpe/main.py

```diff
@@ -159,3 +359,27 @@ def calculate_bleu(actual: str | None, reference: str, max_order: int = 3) -> fl
     :param max_order: max length of n-gram to consider for comparison
     :return: value of BLEU metric
     """
+    if not (
+        isinstance(actual, str) and
+        isinstance(reference, str) and
+        isinstance(max_order, int)
+    ):
+        return None
+
+    precisions = []
+    for order in range(max_order):
```

Comment:
consider passing start argument in `range` instead of adding `1` in next two lines

Reply:
fixed